nixos/modules/services/monitoring/timescale-prometheus.nix

```diff
@@ -0,0 +1,272 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.timescale-prometheus;
+
+in
+{
+
+  options = {
+    services.timescale-prometheus = {
+
+      enable = mkEnableOption "timescale prometheus service";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.timescale-prometheus;
+        defaultText = "pkgs.timescale-prometheus";
+        description = "The timescale-prometheus package to use.";
+      };
+
+      async-acks = mkOption {
```

Comment:
Purely a style thing, feel free to ignore, but I'm curious why you chose `-` instead of camel case when almost every NixOS module uses camel case.